scripts/LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-Copyright 2023 The Parca Authors
+Copyright 2022 The Parca Authors
```

Comment:
Yes, let's keep 2022 for now, the check is not flexible, here is the related issue Lucas-C/pre-commit-hooks#23 and someone opened a PR today!